src/encode/json.rs

```diff
@@ -163,6 +168,49 @@ impl Deserialize for JsonEncoderDeserializer {
     }
 }
 
+#[cfg(feature = "log_kv")]
+mod kv {
+    use log::kv::VisitSource;
+
+    pub fn get_attributes(source: &dyn log::kv::Source) -> anyhow::Result<Vec<(String, String)>> {
```

Comment:
Please document pub fns

Reply:
I have marked this function as `pub(crate)` since it's not relevant to users.